sql/open_postcode_geo-test.sql

```diff
@@ -0,0 +1,12 @@
+-- Test data, one row.
+-- See README for getting the full data.
+DROP TABLE IF EXISTS `civicrm_open_postcode_geo_uk`;
+CREATE TABLE `civicrm_open_postcode_geo_uk` (
+  `postcode` char(8) NOT NULL PRIMARY KEY,
+  `latitude` decimal(9,6) DEFAULT NULL,
+  `longitude` decimal(9,6) DEFAULT NULL,
+  `postcode_no_space` char(7) NOT NULL,
+  UNIQUE KEY `postcode_no_space` (`postcode_no_space`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
```

Comment:
This charset should be utf8mb4 (or not specified) or else if someone tries to join off this is will be unindexed. I don't think mysql will mangle any charactieds even through it differs from the source charset - I don't know if char joins fine to varchar but I'm guessing it does